src/app/components/growing-list/growing-list.component.ts

```diff
@@ -95,26 +127,7 @@ export class GrowingListComponent implements OnInit, OnDestroy {
             .subscribe(deletedObject => this.updateQueue.next(() => this.onDeletedObjectQueued(deletedObject.deleted)));
     }
 
-    ngOnDestroy(): void {
-        this.updateQueueSubscription?.unsubscribe();
-        this.newItemsSubscription?.unsubscribe();
-        this.changedItemsSubscription?.unsubscribe();
-        this.deletedItemsSubscription?.unsubscribe();
-    }
-
-    ngOnChanges(changes: SimpleChanges): void {
-        if (changes.apiLink && changes.apiLink.currentValue) {
-            this.replaceApiLink(changes.apiLink.currentValue);
-        }
-    }
-
     replaceApiLink(newApiLink: ApiLink): void {
-        if (newApiLink.href === this.startApiLink?.href) {
```

Comment:
The plugin lists in the plugin-sidebar didn't get updated when the plugin filter changed. I tried to update them by setting the ApiLink again to the same value, but this `if` prevented the update, since the ApiLink doesn't actually change. However, since this solution didn't really work anyways, I now solved it differently.